# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/extensions/ExtensionPopups.jsm
# Commit: 46c9b479034f
# Full Hash: 46c9b479034f2c6ee710d8d5a703de4a07e77557
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-12 21:43:58
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r 4689b2870416 -r 46c9b479034f browser/components/extensions/ExtensionPopups.jsm
--- a/browser/components/extensions/ExtensionPopups.jsm	Fri Jul 12 14:14:22 2019 +0000
+++ b/browser/components/extensions/ExtensionPopups.jsm	Fri Jul 12 14:14:22 2019 +0000
@@ -293,7 +293,6 @@
     if (this.extension.remote) {
       browser.setAttribute("remote", "true");
       browser.setAttribute("remoteType", E10SUtils.EXTENSION_REMOTE_TYPE);
-      browser.setAttribute("renderroot", "content");
     }
 
     // We only need flex sizing for the sake of the slide-in sub-views of the